.gitattributes

```diff
@@ -0,0 +1 @@
+* -text

```

Comment:
what's this for?

Reply:
https://github.com/microsoft/mu_devops/commits/main/.sync/git_templates/gitattributes_template.txt

it should help automatic prevent line ending conversion